prowler/providers/kubernetes/services/kubelet/kubelet_manage_iptables/kubelet_manage_iptables.py

```diff
@@ -0,0 +1,24 @@
+from prowler.lib.check.models import Check, Check_Report_Kubernetes
+from prowler.providers.kubernetes.services.kubelet.kubelet_client import kubelet_client
+
+
+class kubelet_manage_iptables(Check):
+    def execute(self) -> Check_Report_Kubernetes:
+        findings = []
+        for cm in kubelet_client.kubelet_config_maps:
+            report = Check_Report_Kubernetes(self.metadata())
+            report.namespace = cm.namespace
+            report.resource_name = cm.name
+            report.resource_id = cm.uid
+            if "makeIPTablesUtilChains" not in cm.kubelet_args:
+                report.status = "MANUAL"
+                report.status_extended = f"Kubelet does not have the argument `makeIPTablesUtilChains` in config file {cm.name}, verify it in the node's cm.kubelet_args."
+            else:
+                if cm.kubelet_args["makeIPTablesUtilChains"]:
```

Comment:
This is redundant, in line 13 you are checking if `makeIPTablesUtilChains` is present in `cm.kubelet_args`.

Reply:
no, here I check if it is True, above if the argument exists